test/functional/http/post/0.tx.js

```diff
@@ -0,0 +1,161 @@
+'use strict';
+
+var node = require('../../../node');
+var shared = require('./shared');
```

Comment:
this is inconsistent. in some files we load by .js, in others we are loading without .js

Please stick to the standard format of loading with `.js` for anything that is not a module.